ppgan/models/dc_gan_model.py

```diff
@@ -0,0 +1,142 @@
+#   Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserve.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import paddle
+from paddle import fluid
```

Comment:
paddlegan need to use >= PaddlePaddle 2.0rc,  should not use fluid API

Reply:
I found it in 2.0.0rc document.
![image](https://user-images.githubusercontent.com/14209812/99386303-37a7f580-290d-11eb-821f-4136716b6bc2.png)

Reply:
Oh, Thanks for feedback. There is something wrong with Doc.

2.0 API, there is no fluid.

`squeeze` should be `paddle.squeeze`, other APIs are similar